components/ide-metrics-api/metrics.proto

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+syntax = "proto3";
+
+package ide_metrics_api;
+
+import "google/api/annotations.proto";
+
+option go_package = "github.com/gitpod-io/gitpod/ide-metrics/api";
+option java_package = "io.gitpod.ide_metrics.api";
+
+service MetricsService {
+    rpc AddCounter(AddCounterRequest) returns (AddCounterResponse) {
+        option (google.api.http) = {
+            post: "/reportMetric/{name}",
+            body : "*"
+        };
+    }
+    rpc ObserveHistogram(ObserveHistogramRequest) returns (ObserveHistogramResponse) {
+        option (google.api.http) = {
+            post: "/observeHistogram/{name}/{value}",
+            body : "*"
+        };
+    }
+}
+
+message AddCounterRequest {
+    string name = 1;
+    map<string, string> labels = 2;
```

Comment:
Optional value, if omitted then 1

Reply:
I meant to add:
```
uint64 value = 3;
```

We don't need to add any markers in proto file, but if request does not have this property on server side then we consider value to be 1.